ncm-spma/src/main/pan/components/spma/yum/schema.pan

```diff
@@ -37,6 +37,9 @@ type component_spma_yum = {
     "run" ? legacy_binary_affirmation_string # Run the SPMA after configuring it
     "userpkgs_retry" : boolean = true
     "userpkgs" ? legacy_binary_affirmation_string # Allow user packages
+    @{regexp pattern to install only matching (unescaped) package names.
+        When userpkgs is not defined, it runs as if userpkgs is true.}
```

Comment:
Which makes me wonder, what happens if `userpkgs` is defined and equal to "no"? Does running with `filter` cause it to remove everything? (which would seem inefficient 😀)

Reply:
yes. i can also add explicitly add it to the annotation, but the component does as instructed.

Reply:
Oh, I missed and important bit - this introduces a new component via a template only change:
```
"/software/components/spmalight" = value(...
```

Yeah, a light.pan along with a few lines of explanation at the top would be the best way to handle describing it.